cypress/component/DarCollectionTable/admin_actions.spec.js

```diff
@@ -1,49 +1,71 @@
 /* eslint-disable no-undef */
 import { React } from 'react';
 import { mount } from 'cypress/react';
+import { cloneDeep } from 'lodash/fp';
 import AdminActions from '../../../src/components/dar_collection_table/AdminActions';
 
-const collectionId = 1;
-const props = {
-  collection: {
-    darCollectionId: collectionId,
-    dars: {
-      1: {
-        elections: {
-          1: {status: "Open"}
-        }
-      },
-      2: {
-        elections: {
-          2: {status: "Closed"}
-        }
-      }
-    }
-  },
-  showCancelModal: () => {},
-  updateCollections: () => {},
+const darColl = {
+  'darCollectionId': 2345,
+  "referenceIds": [
```

Comment:
Codacy has a fix for the issue: [Strings must use singlequote.](https://app.codacy.com/gh/DataBiosphere/duos-ui/pullRequest?prid=9874341)
```suggestion
  'referenceIds': [
```